src/plus/subscription/serverConnection.ts

```diff
@@ -12,12 +12,20 @@ import type { DeferredEvent, DeferredEventExecutor } from '../../system/event';
 import { promisifyDeferred } from '../../system/event';
 
 export const AuthenticationUriPathPrefix = 'did-authenticate';
+// TODO: What user-agent should we use?
+const userAgent = 'Visual-Studio-Code-GitLens';
```

Comment:
Just noting this TODO here. Looks like it's a pre-existing TODO from before that was moved here. Not sure if necessary to resolve before this is merged.

Reply:
I'll have to follow-up with Eric on that, but its a pretty old TODO from where it came from and not really needed to solve now.